plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -101,6 +103,7 @@ func NewMetricsReader(logger *zap.Logger, cfg config.Configuration) (*MetricsRea
 		callsMetricName:   buildFullCallsMetricName(cfg),
 		latencyMetricName: buildFullLatencyMetricName(cfg),
 		operationLabel:    operationLabel,
+		tracer:            jtracer.New(),
```

Comment:
@yurishkuro Should I be able to pass it through func NewMetricsReader(...tracer trace.Trace) ?

Reply:
yes

Reply:
you can pick another component to validate the jtracer changes, there are components where JTracer is already passed in.

Reply:
Sorry! I couldn't get that. Can I use `GetLatencies(...)` method to initiate that?

Reply:
I mean use try to change other components than metricsstore.

Reply:
For example, you're passing JTracer here: `cmd/query/app/grpc_handler.go`, but later extracting OT Tracer from it - why not change that path to use OTEL TracerProvider instead?